tests/pytorch/nightly/mnist.libsonnet

```diff
@@ -96,11 +96,19 @@ local utils = import 'templates/utils.libsonnet';
       modelDir: null,
     },
   },
+  local pjrt_ddp = {
+    modelName+: '-ddp',
+    command+: [
+      '--ddp',
+      '--pjrt_distributed',
+    ],
+  },
 
   configs: [
     mnist + convergence + v2_8 + timeouts.Hours(1) + mixins.Experimental,
     mnist + convergence + v2_8 + timeouts.Hours(1) + tpuVm + mixins.Experimental,
-    mnist + convergence + v2_8 + timeouts.Hours(1) + pjrt + mixins.Experimental,
+    mnist + convergence + v2_8 + timeouts.Hours(1) + pjrt,
```

Comment:
do we want to add mnist conv test back? Would resnet be enough?

Reply:
I added this one back since it's our only e2e test that works on TPU v2 that I know of. I can take it off of the schedule if there's another test that covers PJRT + TPU v2.

Reply:
we can keep it.